store/sqlstore/upgrade.go

```diff
@@ -803,3 +808,81 @@ func upgradeDatabaseToVersion524(sqlStore SqlStore) {
 		saveSchemaVersion(sqlStore, VERSION_5_24_0)
 	}
 }
+
+func upgradeDatabaseToVersion525(sqlStore SqlStore) {
+	//if shouldPerformUpgrade(sqlStore, VERSION_5_24_0, VERSION_5_25_0) {
+	saveSchemaVersion(sqlStore, VERSION_5_25_0)
+	//}
+}
+
+func upgradeDatabaseToVersion526(sqlStore SqlStore) {
+	//if shouldPerformUpgrade(sqlStore, VERSION_5_25_0, VERSION_5_26_0) {
+	err := precheckMigrationToVersion526(sqlStore)
+	if err != nil {
+		mlog.Error("Error upgrading DB schema to 5.26.0", mlog.Err(err))
+		os.Exit(EXIT_GENERIC_FAILURE)
+	}
```

Comment:
I am channeling my inner @streamer45 here. We can shorten this to the `if err := fn(); err != nil {}` format. :)